CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 4.0.0 (Dec 23, 2020)
+- Added support of PHP 8.
+- Dropped support of PHP 5.6.
```

Comment:
Add:
- Support from 7.4 (lower in 3.*.*)
- Response serialisation throws exceptions on failure

Reply:
Done